pkg/installmanager/installmanager.go

```diff
@@ -383,6 +384,14 @@ func (m *InstallManager) Run() error {
 	}
 
 	m.log.Info("provisioning cluster")
+
+	if waitForInstallCompleteExecutions, ok := cd.Annotations[constants.WaitForInstallCompleteExecutionsAnnotation]; ok {
+		m.waitForInstallCompleteExecutions, err = strconv.Atoi(waitForInstallCompleteExecutions)
+		if err != nil {
+			m.log.WithError(err).Error("error fetching additional wait-for install-complete executions")
```

Comment:
Include the value of the annotation in the log message.
```suggestion
			m.log.WithField("value", waitForInstallCompleteExecutions).WithError(err).Error("error fetching additional wait-for install-complete executions")
```